### PR TITLE
Release notes for Foreman 3.3.0

### PR DESCRIPTION
**Pull request by upadhyeammit**
Cherry-pick into:
None

### CODE REVIEW

**Comment by melcorr** [comment]
@AkshayGadhaveRH can you let me know if you have any blockers preventing you from completing this review? Thanks!

**Comment by AkshayGadhaveRH** [comment]
HI @melcorr @upadhyeammit sorry for the delay. Somehow I missed the highlighted mail. I have made some suggestions and will go through this once again.

**Comment by melcorr** [comment]
I don't see any suggestions here?

File: guides/doc-Release_notes/topics/foreman-3.3.0.adoc [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+* Add a reports tab to the hosts details page - https://projects.theforeman.org/issues/34782[#34782]
+* Add developer documentation for pagelets - https://projects.theforeman.org/issues/34544[#34544]
+* Include the host comments to a host list page - https://projects.theforeman.org/issues/34516[#34516]
+* Host list page should show the owner of the host - https://projects.theforeman.org/issues/34515[#34515]
+
+=== Settings
+
+* Fix the SettingValueException error message - https://projects.theforeman.org/issues/34921[#34921]
+* Settings - Issues with search - https://projects.theforeman.org/issues/34866[#34866]
+* "Sync Connect Timeout" settings takes invalid value and shows update successful but doesnt reflect the change for invalid valuest reflect the change for invalid values - https://projects.theforeman.org/issues/34838[#34838]
```
Comment (AkshayGadhaveRH) [inline guides/doc-Release_notes/topics/foreman-3.3.0.adoc:18]: "Sync Connect Timeout" settings takes" should be "Sync Connect Timeout" setting takes".

File: guides/doc-Release_notes/topics/foreman-3.3.0.adoc [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+* Deprecate other then DSL setting definitions - https://projects.theforeman.org/issues/34603[#34603]
```
Comment (AkshayGadhaveRH) [inline guides/doc-Release_notes/topics/foreman-3.3.0.adoc:19]: "Deprecate other then" should be "Deprecate other than".

File: guides/doc-Release_notes/topics/foreman-3.3.0.adoc [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+* GraphQL is using Setting model directly - https://projects.theforeman.org/issues/34328[#34328]
+* Initialize Setting for tests without fixtures - https://projects.theforeman.org/issues/33782[#33782]
+* Move setting value collections to SettingPresenter - https://projects.theforeman.org/issues/30861[#30861]
+
+=== DB migrations
+
+* Pending migrations check in seeds does not take plugins into consideration - https://projects.theforeman.org/issues/34913[#34913]
```
Comment (AkshayGadhaveRH) [inline guides/doc-Release_notes/topics/foreman-3.3.0.adoc:26]: "check in seeds does not" should be "check-in seeds do not".

File: guides/doc-Release_notes/topics/foreman-3.3.0.adoc [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+
+=== Templates
+
+* Use foreman request address in windows iPXE  - https://projects.theforeman.org/issues/34710[#34710]
+* Harmonize preseed templates - https://projects.theforeman.org/issues/34658[#34658]
+* Ensure that the insights snippet is being called by honoring the value of host_registration_insights parameter - https://projects.theforeman.org/issues/34525[#34525]
+* AutoYaST PXE templates fail to render if http-proxy parameter is set without http-proxy-port - https://projects.theforeman.org/issues/34489[#34489]
+* Allow puppet setup to be skipped even if you set a puppet master - https://projects.theforeman.org/issues/34388[#34388]
+* AutoYaST SLES template invalid for SLES 15 SP3 - https://projects.theforeman.org/issues/34311[#34311]
+* Support unattented partitionning using crypto under Debian - https://projects.theforeman.org/issues/34307[#34307]
```
Comment (AkshayGadhaveRH) [inline guides/doc-Release_notes/topics/foreman-3.3.0.adoc:97]: "Support unattented partitionning" should be "Support unattended partitioning".

File: guides/doc-Release_notes/topics/foreman-3.3.0.adoc [hunk 5]
```diff
@@ -0,0 +1,266 @@
...
+* Disable EFI local chainloading by default - https://projects.theforeman.org/issues/34532[#34532]
+
+=== Database
+
+* Undefined method format_errors' when db:seed failformat_errors when db:seed fail when db:seed fail - https://projects.theforeman.org/issues/34513[#34513]
+* Ignore dynflow tables for schema dump - https://projects.theforeman.org/issues/33660[#33660]
+
+=== Users, Roles and Permissions
+
+* Infer of permission name for isolated namespace controllers doesnt work - https://projects.theforeman.org/issues/34506[#34506]
```
Comment (AkshayGadhaveRH) [inline guides/doc-Release_notes/topics/foreman-3.3.0.adoc:151]: "doesnt work" should be "doesn't work".

File: guides/doc-Release_notes/topics/foreman-3.3.0.adoc [hunk 6]
```diff
@@ -0,0 +1,266 @@
...
+
+* foreman.rpm pulls in nodejs - https://projects.theforeman.org/issues/34507[#34507]
+* Installer uses hostname, but that package can be absent - https://projects.theforeman.org/issues/34468[#34468]
+
+== SELinux
+
+=== Plugins
+
+* Drop foreman_docker compatibility - https://projects.theforeman.org/issues/34730[#34730]
+* Syncing the git templates on RHEL8 raise SElinux errors - https://projects.theforeman.org/issues/34726[#34726]
```
Comment (AkshayGadhaveRH) [inline guides/doc-Release_notes/topics/foreman-3.3.0.adoc:245]: "RHEL8 raise SElinux" should be "RHEL8 raises SElinux".

**Review by AkshayGadhaveRH — CHANGES_REQUESTED** [review]

**Comment by AkshayGadhaveRH** [comment]
Hey @melcorr @upadhyeammit are you able to see the suggestions now?

**Comment by melcorr** [comment]
@AkshayGadhaveRH yes, thanks!

**Comment by upadhyeammit** [comment]
@AkshayGadhaveRH thanks for suggestions. I have updated the notes and Redmine tickets as per that. Request to review new changes.

**Comment by upadhyeammit** [comment]
@AkshayGadhaveRH I am merging this now as Foreman 3.3 is out and website should reflect that. In case you have any more suggestions then request to raise the pr for same.

**Comment by maximiliankolb** [comment]
@upadhyeammit Would you mind raising another PR to set 3.3 as "stable" instead of RC?

**Pull request merged**